.ldrelease/publish-circleci.sh

```diff
@@ -4,7 +4,7 @@
 set -ev
 
 sudo apt install curl
```

Comment:
Locally this complains that java is not installed and broke the script because of the JDK install prompt. I hope on in the docker Ubuntu image this does not happen.

Reply:
Can we test this in the container locally before merging?